airflow/contrib/kubernetes/kubernetes_request_factory/kubernetes_request_factory.py

```diff
@@ -213,6 +213,18 @@ def extract_dnspolicy(pod, req):
         if pod.dnspolicy:
             req['spec']['dnsPolicy'] = pod.dnspolicy
 
+    @staticmethod
+    def extract_prestop_sleep_time(pod, req):
+        if pod.prestop_wait_time > 0:
+            req['spec']['containers'][0]['lifecycle'] = {'preStop':
+                                                             {'exec':
+                                                                  {'command': ["/bin/sh",
+                                                                               "-c",
+                                                                               "sleep {}".format(pod.prestop_wait_time)]
```

Comment:
No need for shell here:
```suggestion
                                                                  {'command': ["/bin/sleep", str(pod.prestop_wait_time)]
```